src/EditorFeatures/Core.Wpf/InlineDiagnostics/InlineDiagnosticsTaggerProvider.cs

```diff
@@ -31,18 +31,25 @@ namespace Microsoft.CodeAnalysis.Editor.InlineDiagnostics
     internal class InlineDiagnosticsTaggerProvider : AbstractDiagnosticsAdornmentTaggerProvider<InlineDiagnosticsTag>
     {
         private readonly IEditorFormatMap _editorFormatMap;
+        private readonly IClassificationFormatMapService _classificationFormatMap;
+        private readonly IClassificationTypeRegistryService _classificationTypeRegistryService;
+
         protected sealed override IEnumerable<PerLanguageOption2<bool>> PerLanguageOptions => SpecializedCollections.SingletonEnumerable(InlineDiagnosticsOptions.EnableInlineDiagnostics);
 
         [ImportingConstructor]
         [Obsolete(MefConstruction.ImportingConstructorMessage, error: true)]
         public InlineDiagnosticsTaggerProvider(
             IThreadingContext threadingContext,
-            IEditorFormatMapService editorFormatMapService,
             IDiagnosticService diagnosticService,
-            IAsynchronousOperationListenerProvider listenerProvider)
+            IAsynchronousOperationListenerProvider listenerProvider,
+            IEditorFormatMapService editorFormatMapService,
+            IClassificationFormatMapService classificationFormatMap,
```

Comment:
Maybe `classificationFormatMapService` for consistency

Reply:
ye I was originally trying to pass the map isntead of the service and forgot to change it, changed